### DISCUSSION TITLE
九月更新

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1500] [通知]
## 更改记录

- 自述文件勘误 @liubanlaobanzhang (#1471)

## 🧰 管理与维护

- 更新证书 @Richasy (#1499)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2209.1.0'>九月更新</a>.</em>

**Comment by liubanlaobanzhang** [comment]
有个问题……<br>作者打算弃坑了吗

**Comment by windelf0258369** [comment]
請問過期的安全憑證怎麼讓他更新?
